.gitignore

```diff
@@ -156,6 +156,9 @@ docs/opt-out.mlir
 docs/out.mlir
 docs/ret.xdsl
 
+# Type stubs
```

Comment:
```suggestion
# Type stubs
# These are generated type stubs, we do not otherwise use type stubs in the project
```

Reply:
Good point! But as discussed, we will try to commit the stubs, until we find something more satisfactory for Pyright

Reply:
(more like a self-note)

Reply:
I'm actually more for the current state I'm offering; aka explicitly generating stub before pyright when we want to be sure. I really don't like committing autogenerated stuff if I can avoid it.
So, adding this clearer comment :yum: